static/js/i18n/nb_NO.json

```diff
@@ -0,0 +1,21 @@
+{
+    "page-title": "Nåværende",
```

Comment:
In the other i18n translations, 2 space indentations are used. I don't mind either way, but we should keep it consistent.

Reply:
Interesting problem.  These files come from Weblate, and its generator is not configurable as far as I can see.
So, I think we need to change our data files to match the Weblate ones.